cw-plus/cw20/spec.md

```diff
@@ -0,0 +1,144 @@
+---
+order: 1
+---
+
+# Spec: Fungible Tokens
+
+CW20 is a specification for fungible tokens based on CosmWasm.
+The name and design is loosely based on Ethereum's ERC20 standard,
+but many changes have been made. The types in here can be imported by
+contracts that wish to implement this  spec, or by contracts that call
+to any standard cw20 contract.
+
+The specification is split into multiple sections, a contract may only
+implement some of this functionality, but must implement the base.
+
+::: tip
+This section contains CW20 spec implementation details.
+If you are around to play with the contract,
+skip this section and go to the [next page](./setup.md)
+:::
+
+## Base
```

Comment:
This is cut-and-paste from the README, right?

That is fine (and nit to publicize this more), but we need to note to update this if I update the spec (add iterator to list all accounts for example)

Reply:
Then we make sure documentation pipeline is attentive.